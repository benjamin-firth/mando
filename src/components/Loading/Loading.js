import React from 'react';

const Loading = () => {
  return (
    <main>
      <div class="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
    </main>
  )
}

export default Loading;
