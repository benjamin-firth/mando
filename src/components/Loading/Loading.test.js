import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading.js';

describe('Loading', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<Loading />);

    expect(wrapper).toMatchSnapshot();
  })
})
