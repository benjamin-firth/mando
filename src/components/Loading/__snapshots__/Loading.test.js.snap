// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot 1`] = `
<main>
  <div
    class="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</main>
`;
